Applications/DataExplorer/Base/RecentFiles.cpp

```diff
@@ -22,12 +22,12 @@ RecentFiles::RecentFiles(  QObject* parent, const char* slot, QString settingsNa
     : QObject(parent), _settingsName(settingsName)
 {
     _filesMenu = new QMenu(tr("Recent files"));
-    for (int i = 0; i < _maxFiles; i++)
+    for (auto& _fileAction : _fileActions)
```

Comment:
local var starting with underscore.
But it's very neat, that this code conversion was done automatically.